packages/@navikt/side-menu/src/StatefulSideMenu.tsx

```diff
@@ -2,12 +2,13 @@ import * as React from 'react';
 import SideMenu, { Link } from './index';
 
 interface StatefulSideMenuProps {
-    heading: string;
+    heading?: string;
     links: Link[];
     onClick: (index: number) => void;
+    arrowTheme?: boolean;
```

Comment:
Tenker at vi bør definere denne som en custom type for å gjøre det enklere å legge på nye themes senere?